src/button/button-group.scss

```diff
@@ -7,37 +7,29 @@
   align-items: center;
   isolation: isolate;
 
-  $childSelector: 'input, button'; // target nested inputs and buttons
+  $childZindexSelector: '.iui-input-container, .iui-button, .iui-input:where(:not(.iui-input-container .iui-input))';
+  $childBorderSelector: '.iui-input, .iui-button:not(.iui-borderless)';
+  $disabledSelector: '&:disabled, &.iui-disabled, &[aria-disabled="true"]';
```

Comment:
need examples with disabled elements in button-group

Reply:
we already have some
![image](https://user-images.githubusercontent.com/9084735/152866122-bcc756d4-3032-46e9-b304-a6e6359281b7.png)

Reply:
I meant examples with mixed group (buttons + inputs)

Reply:
Added